packages/babel-plugin-transform-react-jsx/test/fixtures/nextReactClassic/does-not-add-source-self-windows/exec.js

```diff
@@ -1,31 +0,0 @@
-var actual = transform(
```

Comment:
I merged this test to the one that runs on linux. Also, I made it input/output rather than calling `transform` in `exec.js`.